spec.html

```diff
@@ -204,11 +204,14 @@ <h1>type</h1>
     </emu-clause>
 
     <emu-clause id="sec-primitive-value">
-      <h1>primitive value</h1>
+      <h1>primitive values and null</h1>
```

Comment:
`null` is a primitive value.
```suggestion
      <h1>primitive value</h1>
```